cli/command/container/diff_test.go

```diff
@@ -0,0 +1,95 @@
+package container
+
+import (
+	"context"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/docker/cli/internal/test"
+	"github.com/docker/docker/api/types/container"
+	"github.com/pkg/errors"
+	"gotest.tools/v3/assert"
+)
+
+func TestRunDiff(t *testing.T) {
+	cli := test.NewFakeCli(&fakeClient{
+		containerDiffFunc: func(
+			ctx context.Context,
+			containerID string,
+		) ([]container.FilesystemChange, error) {
+			return []container.FilesystemChange{
+				{
+					Kind: 0, // Modified ("C")
```

Comment:
We can probably use the types that are defined in the `api/types/container` package for this; https://github.com/docker/cli/blob/baa7a9f7ad378204e8d3ca52121271402ed38a66/vendor/github.com/docker/docker/api/types/container/change_types.go#L4-L9